libs/sr/db/src/Database/DataEntries/DisplayRelicEntry.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  clamp,
+  validateArr,
+  validateObject,
+} from '@genshin-optimizer/common/util'
+import type {
+  LocationKey,
+  RelicMainStatKey,
+  RelicRarityKey,
+  RelicSetKey,
+  RelicSlotKey,
+  RelicSubStatKey,
+} from '@genshin-optimizer/sr/consts'
+import {
+  allLocationKeys,
+  allRelicRarityKeys,
+  allRelicSetKeys,
+  allRelicSlotKeys,
+  allRelicSubStatKeys,
+} from '@genshin-optimizer/sr/consts'
+
+import { DataEntry } from '../DataEntry'
+import type { SroDatabase } from '../Database'
+
+export const relicSortKeys = [
+  'rarity',
+  'level',
+  'relicsetkey',
+  'efficiency',
+  'mefficiency',
+  'probability',
+] as const
+export type RelicSortKey = (typeof relicSortKeys)[number]
+
+export type FilterOption = {
+  relicSetKeys: RelicSetKey[]
+  rarity: RelicRarityKey[]
+  levelLow: number
+  levelHigh: number
+  slotKeys: RelicSlotKey[]
+  mainStatKeys: RelicMainStatKey[]
+  substats: RelicSubStatKey[]
+  locations: LocationKey[]
+  showEquipped: boolean
+  showInventory: boolean
+  locked: Array<'locked' | 'unlocked'>
+  rvLow: number
+  rvHigh: number
```

Comment:
`rvLow, rvHigh` is a filtering metric:
![image](https://github.com/frzyc/genshin-optimizer/assets/1754901/14639b2a-d5ee-4b41-9741-c6783029577e)

Reply:
Yeah that is a good question. I'd probably just put a `// TODO:` here until we figure that out.